libsignal-protocol/src/crypto/mod.rs

```diff
@@ -398,3 +399,46 @@ unsafe extern "C" fn internal_cipher(
         Err(e) => e.code(),
     }
 }
+
+#[cfg(all(test, feature = "crypto-native", feature = "crypto-openssl"))]
+mod crypto_tests {
+    use super::*;
+
+    #[test]
+    fn test_crypter() {
```

Comment:
to run this test
use 
```bash
RUST_BACKTRACE=1 cargo test --no-fail-fast --features="crypto-native,crypto-openssl" test_crypter
```

Reply:
Can you move the `#[cfg(all(feature = "crypto-native", feature = "crypto-openssl"))]` to above the test? That way it's saying *this* test requires both native and OpenSSL crypto.